diffusion_with_autoscaler/autoscaler.py

```diff
@@ -554,7 +558,7 @@ def scale(self, replicas: int, metrics: dict) -> int:
                 scale_out_interval=10,
                 scale_in_interval=10,
                 max_batch_size=8,  # for auto batching
-                timeout_batching=1,  # for auto batching
+                max_batch_delay=1,  # for auto batching
```

Comment:
batch_wait_interval would be slightly more explicit and consistent with the arguments above.